pkg/lang/python/imports.go

```diff
@@ -40,13 +40,10 @@ type Attribute struct {
 
 type Imports map[string]Import
 
-func (imp Import) ModuleDir() string {
-	moduleRoot := ""
-	if imp.ParentModule != "" {
-		moduleRoot = imp.ParentModule
-	}
+func (imp Import) FullyQualifiedModule() string {
+	moduleRoot := imp.ParentModule
 	if imp.Name != "" {
-		if moduleRoot != "" {
+		if moduleRoot != "" && !strings.HasSuffix(moduleRoot, ".") {
```

Comment:
Is "suffix" here really just a stand-in for "only consists of", or is it actually a suffix, and there may be non-`.`s in there too?

A comment here might be nice, because imo it's pretty easy to get lost in what's-doing-what of all this. But mostly I'm just asking for my own learning.